pkg/kubelet/qos/policy_test.go

```diff
@@ -171,6 +172,7 @@ type oomTest struct {
 	memoryCapacity  int64
 	lowOOMScoreAdj  int // The max oom_score_adj score the container should be assigned.
 	highOOMScoreAdj int // The min oom_score_adj score the container should be assigned.
+	expectedErr     string
```

Comment:
Checking error value is a bit fragile in my opinion unless we really want to control all changes to an error message. `expectError bool `should be enough.